src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -366,4 +366,25 @@
   <data name="net_http_feature_UWPClientCertSupportRequiresCertInPersonalCertificateStore" xml:space="preserve">
     <value>Client certificate was not found in the personal (\"MY\") certificate store. In UWP, client certificates are only supported if they have been added to that certificate store.</value>
   </data>
+  <data name="net_http_content_unexpected_end" xml:space="preserve">
+    <value>Encountered an unexpected end of the content stream.</value>
+  </data>
+  <data name="net_http_headers_missing_location" xml:space="preserve">
+    <value>The response code indicates a redirect, but the 'Location' header is missing.</value>
+  </data>
+  <data name="net_http_invalid_chunk_size" xml:space="preserve">
+    <value>The response stream contains an invalid chunk size.</value>
```

Comment:
OK, yea I saw that one, wasn't sure what the practice was on the level of detail we need.